src/lightning_lite/strategies/launchers/multiprocessing.py

```diff
@@ -0,0 +1,174 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
Reviewer note: The base class gets moved, but the implementations need to be copied due to dependencies on PL.